### PR TITLE
GCP how-to-guide MemoryStore

### PR DESCRIPTION
**Pull request by Sid-Lais**

### CODE REVIEW

File: how-to-guides/cloud-services/gcp/connecting-to-memorystore.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+4. Choose the **VPC Network** under the **Set up connection** section.
+5. Setup the service connection policy if prompted.
+6. Fill in the network name and the **private subnet** in the policy section.
+7. After setting up the connection  policy, click on **Create Cluster**.
+
+![image](../../../.gitbook/assets/gcp-memorystore-creation.png)
+
+### Step 2: Accessing Memeorystore from DevBox
+
+1. Go to **Memeorystore > Clusters**.
```
Comment (dray92) [inline how-to-guides/cloud-services/gcp/connecting-to-memorystore.md:49]:
```suggestion
1. Go to **Memorystore > Clusters**.
```

Comment (Sid-Lais) [reply how-to-guides/cloud-services/gcp/connecting-to-memorystore.md]: Fixed

**Review by mishushakov — CHANGES_REQUESTED** [review]
Grammatical

**Comment by dray92** [comment]
@Sid-Lais lint errors

**Comment by Sid-Lais** [comment]
@dray92 Fixed errors

**Pull request merged**